src/Web/WebPush.hs

```diff
@@ -225,6 +227,28 @@ mkPushNotification endpoint p256dh auth =
        , _pnMessage = ()
     }
 
+
+-- |Standard payload for web-pushes (but can be customized by passing
+-- any data to `sendPushNotification`, which has ToJSON instance)
+data PushNotificationMessage = PushNotificationMessage
+    { title :: T.Text
+    , body :: T.Text
+    , icon :: T.Text
+    , url :: T.Text
+    , tag :: T.Text
+    } deriving (Eq, Show)
```

Comment:
Is there should be prefixes for the record fields?

Reply:
back-compatibility